setup.R

```diff
@@ -0,0 +1,33 @@
+#check for present library paths
```

Comment:
Would it be possible to call this from `setup.py`? I would prefer not to need additional commands to install the r dependencies. How does this work with the conda package? Do the requirements listed here also need to be added to `meta.yaml`?